src/sandstorm/package.capnp

```diff
@@ -125,7 +125,7 @@ struct Manifest {
     # never provide a way to make argv[0] contain something other than the executable name, as
     # you can technically do with the `exec` system call.
   }
-
+M
```

Comment:
Hi @audreyt , I think this change is a stray added letter. Is that right? If so, might be best to remove it.